core/src/main/java/com/linecorp/armeria/common/MediaType.java

```diff
@@ -472,6 +472,18 @@ private static MediaType addKnownType(MediaType mediaType) {
 
     public static final MediaType GZIP = createConstant(APPLICATION_TYPE, "x-gzip");
 
+    /**
+     * <a href="https://datatracker.ietf.org/doc/html/rfc7932">Brotli Compression format, a lossless data
+     * compression format.</a>.
+     */
+    public static final MediaType BROTLI = createConstant(APPLICATION_TYPE, "brotli");
```

Comment:
Could you also update `MediaTypeNames` together?

Reply:
Sure, that's now done.